app/controllers/home_controller.rb

```diff
@@ -38,9 +40,10 @@ def static
 
   def unsubscribe
     code = params[:code]
-    User.all.select {|x| x.encrypted_email == code }.each {|x| x.update_attributes subscribed_to_newsletter: false }
-    flash[:message] = "You have been unsubscribed from the newsletter."
-    redirect_to "/anime"
+    User.all.select { |x| x.encrypted_email == code }
+      .each do |x| x.update_attributes subscribed_to_newsletter: false end
```

Comment:
I should change this in rubocop. Since Hummingbird is a big team, having a vague distinction like "do..end for imperative, {} for functional" is probably a bad idea.  I'll switch this back to the standard bbatsov "do..end for multi line, {} for single line"